trento/adoc/trento-mcp-install-systemd.adoc

```diff
@@ -0,0 +1,156 @@
+include::product-attributes.adoc[]
+
+[[sec-trento-mcp-server-systemd]]
+==== systemd {tr_mcp} deployment
+
+A systemd-based installation of the {tr_mcp} using RPM packages can be performed manually on the latest supported versions of {sles4sap}, from 15 SP4 up to 16.0. For installations on service packs other than the current one, make sure to update the repository URL as described in the relevant notes throughout this guide.
+
+include::partials/supported-versions.adoc[]
+
+===== Installing {tr_mcp} using RPM packages
+
+. Install the {tr_mcp} package:
++
+[source,bash]
+----
+zypper install mcp-server-trento
+----
+
+====== Configure {tr_mcp}
+
+. Create the {tr_mcp} configuration file at +/etc/trento/mcp-server-trento+ by copying the example:
++
+[source,bash]
+----
+cp /etc/trento/mcp-server-trento.example /etc/trento/mcp-server-trento
+----
++
+. Edit the configuration file to point to your {trserver}:
++
+[source,bash]
+----
+vim /etc/trento/mcp-server-trento
+----
++
+
+Example configuration:
++
+[source,env]
+----
+AUTODISCOVERY_PATHS=/api/all/openapi,/wanda/api/all/openapi
+ENABLE_HEALTH_CHECK=false
+HEADER_NAME=Authorization
+HEALTH_API_PATH=/api/healthz
+HEALTH_PORT=8080
+# OAS_PATH=http://trento.example.com/api/all/openapi,http://trento.example.com/wanda/api/all/openapi
+PORT=5000
+TAG_FILTER=MCP
+TRANSPORT=streamable
+TRENTO_URL=https://trento.example.com
+VERBOSITY=info
+INSECURE_SKIP_TLS_VERIFY=false
+----
+
+[%hardbreaks]
+Configure the {tr_mcp} using either `TRENTO_URL` or `OAS_PATH`.
+If `OAS_PATH` is left empty, the {tr_mcp} automatically discovers the APIs from the {trserver} using `TRENTO_URL`.
+If `OAS_PATH` is set, it takes precedence and `TRENTO_URL` is ignored.
+
+**Use `TRENTO_URL` when:**
+
+* {trserver} is deployed behind a reverse proxy (NGINX, etc.).
+* The {tr_mcp} runs on a different host or network than {trserver}.
+* You want to use external or public URLs.
+* You prefer automatic API autodiscovery (recommended for production).
+
+Example:
+
+[source,env]
+----
+TRENTO_URL=https://trento.example.com
+PORT=5000
+----
+
+**Use `OAS_PATH` when:**
+
+* The {tr_mcp} runs on the same host as the {tr_web} and {tr_wanda} components.
+* You want to connect directly to internal services without autodiscovery.
+* You need to bypass reverse proxy configurations.
+
+Example:
+
+[source,env]
+----
+OAS_PATH=http://localhost:4000/api/all/openapi,http://localhost:4001/wanda/api/all/openapi
+PORT=5000
```

Comment:
Same as before, I think this example is redundant with the configuration example given previously.